manifests/ntpd.pp

```diff
@@ -1,6 +1,8 @@
 # @summary Sets up NTP configuration variables
 #
-# @param servers The list of NTP time servers for the network.
+# @param servers
+#   **DEPRECATED** Use simp_options::ntp instead.
```

Comment:
We can't require the use of `simp_options::ntp`. Users still need to be able to use the module in a non-SIMP environment.